pkgs/by-name/sp/SpectralSuite/package.nix

```
@@ -0,0 +1,145 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  nix-update-script,
+  cmake,
+  ninja,
+  pkg-config,
+  libx11,
+  libxrandr,
+  libxinerama,
+  libxext,
+  libxcursor,
+  libxtst,
+  freetype,
+  fontconfig,
+  webkitgtk_4_1,
+  curl,
+  alsa-lib,
+  libsysprof-capture,
+  pcre2,
+  util-linux,
+  libselinux,
+  libsepol,
+  libthai,
+  libdatrie,
+  libxdmcp,
+  libdeflate,
+  lerc,
+  xz,
+  libwebp,
+  libxkbcommon,
+  libepoxy,
+  sqlite,
+}:
+
+let
+  juce = fetchFromGitHub {
+    owner = "juce-framework";
+    repo = "JUCE";
+    tag = "8.0.6";
+    hash = "sha256-uwZVBrvb5O9LEh00y93UeEu4u4rd+tLRCdQdxsMpXNg=";
+  };
+in
+
+stdenv.mkDerivation (finalAttrs: {
+  name = "SpectralSuite";
```

```suggestion
  pname = "spectralsuite";
```
1. the versionless part is called `pname`. `name` is composed implicitly as `${pname}-${version}` (except for some scopes, like Python libraries and Vim plugins) and isn't supposed to be set explicitly
2. `pname` should be lowercase all the way, and attrpath and filepath should match it: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming